ParisBNF/abb/BNFabb39.xml

```diff
@@ -39,7 +39,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                           <material key="parchment"/>
                        </support>
                        <extent>
-                          <measure unit="leaf">205+1</measure>
+                          <measure unit="leaf">206 (205 + 1)</measure>
```

Comment:
Hadn't we previously said that this should be encoded as 
`<measure unit="leaf" quantity="206">205+1</measure>`?

Reply:
Ja, stimmt. Danke!